src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -146,8 +146,26 @@ export interface IDetachedXTermOptions {
 	colorProvider: IXtermColorProvider;
 	capabilities?: ITerminalCapabilityStore;
 	readonly?: boolean;
+	processInfo: ITerminalProcessInfo;
 }
 
+export interface IDetachedTerminalInstance extends IDisposable {
```

Comment:
A comment explaining that it emulates an `ITerminalInstance` in order to enable more features would be handy in the future.